pkg/sink/void.go

```diff
@@ -0,0 +1,19 @@
+package sink
+
+import "fmt"
+
+type Void struct {
+	Debug bool
+}
+
+func NewVoid(url string) Void {
+	return Void{true}
```

Comment:
Maybe you could write `Debug: true`

Reply:
I’m lost with go syntax after a couple months writing more rust ahaha thanks! Good catch!

Reply:
hahahaa
It would work but without the `Debug` key it would be less beautiful (IMHO).
You know, too much time reading Python code with `func(True, False, 1, 2)`

Reply:
Yes, it's auto-explainable for new readers... (no need to open the definition to understand the flag, or at least know what's about.